src/Store.lua

```diff
@@ -70,9 +91,22 @@ end
 	Get the current state of the Store. Do not mutate this!
 ]]
 function Store:getState()
+	if self.isDispatching then
```

Comment:
This is a nitpick, but we should probably call this `_isDispatching` to indicate that it's not meant to be read (or, heaven forbid, written) by Rodux users.

Reply:
ok. since Signal ends up using it, I was expressing it as 'module public'. But you're right, I should think about `_` as 'package private'.